Code/GraphMol/MarvinParse/testMrvToMol.cpp

```diff
@@ -122,13 +122,6 @@ void testSmilesToMarvin(const SmilesTest *smilesTest) {
 
       std::string smilesOut = MolToSmiles(*localVars.smilesMol, ps);
 
-      // code to generate the expected files
-
-      // {
```

Comment:
Maybe add that once at the top of the file? so we don't have to reinvent it everytime?

Reply:
I added a function to read the expected results